novuhq/website
# feat: add linearDark and linearLight themes for inbox

**Description**

This PR brings new themes for inbox reusable component







**Steps to test**

1. Open [the page](https://deploy-preview-299--novu-website.netlify.app/inbox/)
2. Confirm that everything looks and works as expected


**Screenshots**

<details>
<summary>LinearDark</summary>

![image](https://github.com/user-attachments/assets/ff261dd6-3c78-4efb-b131-2b589c469340)
</details>

<details>
<summary>LinearLight</summary>

![image](https://github.com/user-attachments/assets/f80e3b2b-3100-4983-aa75-47575a77a546)
</details>
